gradle/libs.versions.toml

```diff
@@ -22,7 +22,7 @@ androidXFragment = "1.2.4"
 androidXLeanback = "1.0.0"
 androidXMetrics = "1.0.0-alpha04"
 androidXMultidex = "2.0.1"
-androidXNavigation = "2.3.0"
+androidXNavigation = "2.8.0"
```

Comment:
Versions before this either contains pretty [significal](https://developer.android.com/jetpack/androidx/releases/navigation#2.8.0-alpha02) problems